sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-oidc/README.md

```diff
@@ -64,15 +70,14 @@ mvn spring-boot:run
 
 ### Validation
 
-1. Access `http://localhost:8080/` as index page.
+1. Access `http://localhost:8080/login` as index page.
 2. Sign up/in.
-3. Access greeting button.
-4. Logout.
-5. Sign in.
-6. Profile edit.
-7. Password reset.
-8. Logout
-9. Sign in.
+3. Logout.
```

Comment:
What's this `logout` for?

Reply:
I guess there are 2 flows in the login page, I will change this description.